cmd/tree-walk.go

```diff
@@ -21,15 +21,21 @@ import (
 	"strings"
 )
 
-// list of all errors that can be ignored in tree walk operation.
-var walkResultIgnoredErrs = []error{
+// list of all errors that can be ignored in tree walk operation in XL
+var xlTreeWalkIgnoredErrs = []error{
 	errFileNotFound,
 	errVolumeNotFound,
 	errDiskNotFound,
 	errDiskAccessDenied,
 	errFaultyDisk,
 }
 
+// list of all errors that can be ignored in tree walk operation in FS
+var fsTreeWalkIgnoredErrs = []error{
+	errFileNotFound,
+	errVolumeNotFound,
+}
+
```

Comment:
It is better these two are declared in fs-v1* xl-v1* files. because if you see tree-walk.go is abstracted from xl or fs